sdk/messaging/azservicebus/internal/constants.go

```diff
@@ -3,5 +3,7 @@
 
 package internal
 
+const ModuleName = "azservicebus"
```

Comment:
This should be the fully qualified module name.

Reply:
Could you elaborate on what the fully qualified module name should be? For context, I was following the module naming for eventgrid: https://github.com/Azure/azure-sdk-for-go/blob/679a930e6edd51f69f811bc1f8296a0dece799fa/sdk/messaging/eventgrid/azeventgrid/internal/version.go#L9

Reply:
It's the value of `module` from the `go.mod` file so `github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus`.

Looks like `azeventgrid` is incorrect. :(

Reply:
Looks like we used to generate/have it one way, and now it goes another. I guess the issue is I haven't regenerated azeventgrid recently enough.

Reply:
Ah, maybe it's not even auto-generated. I note that aztables swapped from the short to the long format, so I probably just missed the memo.